grammars.yml

```diff
@@ -554,6 +554,8 @@ vendor/grammars/language-ncl:
 - source.ncl
 vendor/grammars/language-ninja:
 - source.ninja
+vendor/grammars/language-nunjucks:
+- text.html.nunjucks
```

Comment:
We're missing the `source.nunjucks` entry that is also provided by the grammar. You're going to need to run `script/grammar-compiler` or manually add the other source else the syntax highlighter won't be able to find it.

Reply:
So just that one line needs to be added?

Reply:
Yup.